mesa/examples/basic/boltzmann_wealth_model/model.py

```diff
@@ -10,7 +10,7 @@ class BoltzmannWealthModel(mesa.Model):
     highly skewed distribution of wealth.
     """
 
-    def __init__(self, n=100, width=10, height=10, seed=None):
+    def __init__(self, seed, n=100, width=10, height=10):
```

Comment:
```suggestion
    def __init__(self, seed=None, n=100, width=10, height=10):
```

Reply:
This change was just for testing, removed now